less/index.less

```diff
@@ -4,6 +4,3 @@
 @import "components/index";
 @import "pages/index";
 @import "../node_modules/react-select/less/default.less";
-@import (less) "../node_modules/webmaker-app-icons/css/ionicons.css";
-
-@font-face { font-family: "Ionicons"; src: url("vendor/webmaker-app-icons/fonts/ionicons.eot?v=2.0.1"); src: url("vendor/webmaker-app-icons/fonts/ionicons.eot?v=2.0.1#iefix") format("embedded-opentype"), url("vendor/webmaker-app-icons/fonts/ionicons.ttf?v=2.0.1") format("truetype"), url("vendor/webmaker-app-icons/fonts/ionicons.woff?v=2.0.1") format("woff"), url("vendor/webmaker-app-icons/fonts/ionicons.svg?v=2.0.1#Ionicons") format("svg"); font-weight: normal; font-style: normal; }
```

Comment:
Yep see https://github.com/mozilla/teach.webmaker.org/issues/714#issuecomment-95921298 for more explanation.